src/hotspot/cpu/x86/stubGenerator_x86_64.hpp

```diff
@@ -598,6 +598,9 @@ class StubGenerator: public StubCodeGenerator {
   address generate_cont_returnBarrier();
   address generate_cont_returnBarrier_exception();
 
+  address generate_cont_preempt_stub();
+  address generate_cont_resume_monitor_operation();
```

Comment:
The declaration of `generate_cont_resume_monitor_operation` seems to be unused.

Reply:
Removed.